packages/atomic/src/pages/test.html

```diff
@@ -18,11 +18,13 @@
           accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
           organizationId: 'searchuisamples',
         });
+        searchInterface.executeFirstSearch();
       })();
     </script>
   </head>
+
   <body>
-    <atomic-search-interface id="search" reflect-state-in-url="false">
+    <atomic-search-interface id="search" reflect-state-in-url="true" sample>
```

Comment:
remove "sample"

I think @nrawji you had a reason to disable `reflect-state-in-url` ?